src/content/api/module-methods.md

```diff
@@ -147,6 +149,8 @@ T> Note that all options can be combined like so `/* webpackMode: "lazy-once", w
 
 T> Note that `webpackInclude` and `webpackExclude` options do not interfere with the prefix. eg: `./locale`.
 
+`webpackExports`: tells webpack to only bundle the used exports of a module when using dynamic imports. Can decrease the output size of a chunk. Available since webpack 5.0.0-beta.18
```

Comment:
```suggestion
`webpackExports`: tells webpack to only bundle the used exports of a module when using dynamic imports. It can decrease the output size of a chunk. Available since webpack [5.0.0-beta.18](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.18).
```

Reply:
yup also put webpack into square bracket for a better link